.gitlocalize.yml

```diff
@@ -1,8 +1,18 @@
 pretranslate:
   segments:
+    ignore_frontmatter_keys:
+    - "layout"
+    - "permalink"
+    - "noindex"
+    - "date"
+    - "override"
+    - "codelabs"
+    - "tags"
+    - "authors"
+  
     ignore_patterns:
     - "Native"
-    - "{%\\s?([a-zA-Z0-9\\.\\-\\?\'\"\\s]*)\\s?%}"
+    - !ruby/regexp /{%\s?([a-zA-Z0-9\.\-\?\'\"\,\!\s]*)\s?%}/
```

Comment:
Is this a ruby style regex?

Reply:
Yes, correct. Apart from the !ruby/regexp prefix it's a common regex (with additional escaping)